### DISCUSSION TITLE
Not work with Automatic1111 1.3.2

### DISCUSSION THREAD

**Discussion by suonnon** [huchenlei/sd-webui-openpose-editor#19] [Q&A] [answered]
Hi, I installed this extension on A1111 ver 1.3.2 but after install and fresh startup A1111, the extension does not show on the interface.

**Comment by huchenlei** [comment]
Can you file a bug report and provide more context, e.g. ControlNet version, console logs, etc.

**Reply by suonnon** [reply]
> Can you file a bug report and provide more context, e.g. ControlNet version, console logs, etc.

This is my A1111 UI screen after I installed the extension and relaunch. I can't find the interface of the "sd-webui-openpose-editor" extension. Is it a bug or am I missing something?

![Screenshot 2023-06-11 213145](https://github.com/huchenlei/sd-webui-openpose-editor/assets/133345496/cc1087ad-c76d-4d65-aa8e-6d51fdc41d61)

**Reply by huchenlei** [reply, marked as answer]
See usage section of README. 

The extension does not add a tab or accordion. The main entry point is the edit button on controlnet preview image.

**Reply by suonnon** [reply]
> See usage section of README.
> 
> The extension does not add a tab or accordion. The main entry point is the edit button on controlnet preview image.

Thank you so much for super quick response! I didn't notice that. Btw, you should place the instruction "Edit button on the controlnet preview image" on the README, then it could be easy to follow up the usage I guess.

**Reply by huchenlei** [reply]
I added a video demo to README. Hopefully that will help.

**Reply by suonnon** [reply]
> I added a video demo to README. Hopefully that will help.

Your support is insane!!!! Super quick!!!!
